src/Analyzer.Reports/SarifReportWriter.cs

```diff
@@ -18,13 +18,17 @@ namespace Microsoft.Azure.Templates.Analyzer.Reports
     /// </summary>
     public class SarifReportWriter : IReportWriter
     {
+        /// <summary>
+        /// Class used to output information to the SARIF file
+        /// </summary>
+        public SarifLogger sarifLogger;
```

Comment:
Make this a property instead of a variable.  It's generally best practice to have public members be properties instead of just variables.  We should include this in the contribution guide at some point.
There's not really an ideal place for properties in this class right now - I think I would move it to just below the constructor.

```cs
        public SarifLogger SarifLogger { get; set; }
```